css/index.css

```diff
@@ -112,6 +113,46 @@ p {
     justify-content: center;
 }
 
```

Comment:
Nice work using the parent styles and just overriding the color that changes in each box.